guacamole/src/main/webapp/app/settings/services/preferenceService.js

```diff
@@ -58,7 +58,7 @@ angular.module('settings').factory('preferenceService', ['$injector',
          *
          * @type String
          */
-        inputMethod : null
+        inputMethod : 'none'
```

Comment:
We don't have to do this now, but maybe these should be constants at some point?